incubator/hnc/Makefile

```diff
@@ -352,3 +332,26 @@ endif
 ifeq ($(COULDNT_READ_RELEASE_IMG), 0)
 	$(error The image ${HNC_RELEASE_IMG} already exists. Force and overwrite this image by using HNC_FORCE_RELEASE=true)
 endif
+
+# Generate the Krew manifest and put it in manifests/. Note that 'manifests' must exist because
+# krew-build calls krew-tar calls build calls manifests.
+krew-build: krew-tar
+	cp hack/krew-kubectl-hns.yaml manifests/krew-kubectl-hns.yaml
+	sed -i 's/HNC_KREW_TAR_SHA256/${HNC_KREW_TAR_SHA256}/' manifests/krew-kubectl-hns.yaml
+	sed -i 's/HNC_IMG_TAG/${HNC_IMG_TAG}/' manifests/krew-kubectl-hns.yaml
+	sed -i 's/HNC_RELEASE_REPO_OWNER/${HNC_RELEASE_REPO_OWNER}/' manifests/krew-kubectl-hns.yaml
```

Comment:
Question: what does `'s/HNC_KREW_TAR_SHA256/${HNC_KREW_TAR_SHA256}/'` mean specificially?

Reply:
It means "replace the string `HNC_KREW_TAR_SHA256` with the contents of the variable that happens to have the same name." `sed -i` does an in-place search-and-replace using regexes.